src/utils/getCommentsByArticleId.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+const NCNews = axios.create({
```

Comment:
we can put all our axios requests in one file and then we dont need to instantiate axios again every time we want to create a new api request function